Makefile

```diff
@@ -757,7 +757,7 @@ build/check-doc.build-stamp: doc/build/html/index.html doc/build/html/htmlcov/in
 		| egrep -C1 -e WARNING -e CRITICAL -e ERROR \
 		| egrep -v \
 			-e "warnings.warn\(f'\"{wd.path}\" is shallow and may cause errors'\)" \
-			-e "No such file or directory: '.*.examples'.$$" \
+			-e "No such file or directory: '.*.examples'.( \[docutils\]\s*)?$$" \
```

Comment:
Output format of the warning changed in the latest sphinx version.

Will be working to remove the need for these ignores in a different PR #869